envs/windows/conda/environment-py3.8.yml

```diff
@@ -7,5 +7,5 @@ dependencies:
   - sphinx
   - conda-build
   - conda-verify
-  - arcgis=2.0.0
+  - arcgis=2.0.1

```

Comment:
consider removing the version statement so we are automatically on the latest version like with the other packages